### DISCUSSION TITLE
fr/hub/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#8755] [Docs]
# fr/hub/

Apprends à former et à déployer tes modèles YOLOv5 et YOLOv8 avec Ultralytics HUB . Explore les modèles pré-entraînés, les modèles et les diverses intégrations.

https://docs.ultralytics.com/fr/hub/

**Comment by Hamzaboughfir123** [comment]
can I just upload my trained model here without need to train it again ?

**Reply by glenn-jocher** [reply]
Hey there! 😊 Absolutely, you can upload your trained model without needing to retrain it. If you're using Ultralytics HUB, simply navigate to the "Projects" section, select your project, and you'll find an option to upload your model. Make sure your model is in the correct format supported by HUB. 

For example, if you have a YOLOv8 model trained and saved as `my_model.pt`, you can upload it directly to your project. 

If you encounter any issues or have further questions, feel free to reach out. Happy detecting! 🚀

**Reply by Hamzaboughfir123** [reply]
my model is trained outside hub, there is an option to upload it to hub ?

**Reply by glenn-jocher** [reply]
Hello! 😊

Yes, you can definitely upload a model trained outside of Ultralytics HUB to the platform. Here's how you can do it:

1. **Navigate to the Projects Section**: Once you're logged into Ultralytics HUB, go to the "Projects" section.
2. **Select Your Project**: Choose the project where you want to upload your model.
3. **Upload Your Model**: Look for an option to upload your model. You can upload your trained model file (e.g., `my_model.pt`) directly to the project.

This way, you can leverage the powerful features of Ultralytics HUB for deployment, inference, and further analysis without needing to retrain your model.

If you need any assistance or run into any issues, feel free to reach out. We're here to help! 🚀

**Comment by Hamzaboughfir123** [comment]
I can't see when exactly I can upload my trained model to the ultralytics
HUB ????




[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
10/06/24
17:20:43

Le sam. 8 juin 2024 à 12:37, Glenn Jocher ***@***.***> a
écrit :

**Reply by glenn-jocher** [reply]
@Hamzaboughfir123 hello! 😊

It sounds like you're having some trouble figuring out when you can upload your trained model to Ultralytics HUB. Let's clarify this process for you.

Once you've trained your model, you can upload it to Ultralytics HUB at any time. Here’s a step-by-step guide to help you through the process:

1. **Navigate to the Projects Section**: After logging into Ultralytics HUB, go to the "Projects" section from the sidebar.
2. **Select Your Project**: Choose the project where you want to upload your model. If you don't have a project yet, you can create one directly from the Home page.
3. **Upload Your Model**: Within your selected project, look for an option to upload your model. You can upload your trained model file (e.g., `my_model.pt`) directly to the project.

This way, you can leverage the powerful features of Ultralytics HUB for deployment, inference, and further analysis without needing to retrain your model.

For more detailed instructions, you can refer to the [Ultralytics HUB Quickstart Guide](https://docs.ultralytics.com/hub/quickstart/). This guide will walk you through the entire process, from uploading datasets to training and deploying models.

If you need any further assistance or run into any issues, feel free to reach out. We're here to help! 🚀
